nmostesting/suites/IS1101Test.py

```diff
@@ -1733,29 +1744,31 @@ def test_04_03(self, test):
             """
             The test requires streaming from a Sender in order to
             verify the state of the Receiver and the associated outputs.
-            """
-            """
-            if IS11_RefecenceSender is not configured fail 4.3 with log indicating
-            that IS11_RefecenceSender is not configured
+            if IS11_REFERENCE_SENDER is not configured fail 4.3 with log indicating
+            that IS11_REFERENCE_SENDER is not configured
             """
             if len(CONFIG.IS11_REFERENCE_SENDER) == 0:
-                return test.FAIL("IS11_RefecenceSender is not configured")
+                return test.DISABLED("Please configure IS11_REFERENCE_SENDER in Config.py")
             if len(self.receivers_outputs) == 0:
                 return test.UNCLEAR("No IS-11 receivers support outputs")
             for receiver_id in self.receivers_outputs:
-                _, response = TestHelper.do_request('GET', self.node_url + "receivers/" + receiver_id)
+                valid, response = TestHelper.do_request('GET', self.node_url + "receivers/" + receiver_id)
+                if not valid:
+                    return test.FAIL("Unexpected response from the Node API: {}".format(response))
                 if response.status_code != 200:
                     return test.FAIL("The Node API request for receiver {} has failed: {}"
                                      .format(receiver_id, response.json()))
                 receiver_format = response.json()["format"]
                 receiver_transport_file = response.json()["transport"]
                 if (receiver_format == "urn:x-nmos:format:video"):
                     """
-                    using $IS11_RefecenceSender GET sender_id from N?/senders/*/ where
+                    using $IS11_REFERENCE_SENDER GET sender_id from N?/senders/*/ where
```

Comment:
`$` ?

`N?` ?

Reply:
N? definition is in the test plan, $ is used in test plan to "highlight" variables ... $ should not be used in effective test comments but I think N? shortcuts could be defined once at the beginning of the test file and be used in comments.